src/quo2/components/community/icon.cljs

```diff
@@ -0,0 +1,11 @@
+(ns quo2.components.community.icon
```

Comment:
I moved the community icon to quo2 @flexsurfer 

Removed a lot of the excess code in it, this doesn't handle the "Status" default community icon, but imo we should move away from this community and start using real data as we are adding to many branches to the code just for this community